ami-build/scripts/install-fun.sh

```diff
@@ -0,0 +1,223 @@
+#!/bin/bash -eux
+
+set -o pipefail
+
+
+if [[ -z "$AGENT_VERSION" ]]; then
+   echo "AGENT_VERSION unset, exiting" >&2
+   exit 1
+fi
+
+UNAME_ARCH=$(dpkg --print-architecture)
+if [[ "${UNAME_ARCH}" == "amd64" ]]; then
+    ARCH=x64
+    FULL_ARCH=x86_64
+else
+    ARCH="$UNAME_ARCH"
+    FULL_ARCH="$UNAME_ARCH"
+fi
+export ARCH
+AGENT_FILE="vsts-agent-linux-${ARCH}-${AGENT_VERSION}"
+AGENT_DL_URL="https://vstsagentpackage.azureedge.net/agent/${AGENT_VERSION}/${AGENT_FILE}.tar.gz"
+APT_PKGS_AGENT=(
+    awscli
+    git
+    inotify-tools
+    jq)
+APT_PKGS_BASE=(
+    apt-transport-https
+    ca-certificates
+    gnupg-agent
+    software-properties-common
+    wget)
+APT_PKGS_BUILD=(
+    docker-ce
+    docker-ce-cli
+    containerd.io
+    expect
+    skopeo
+    zstd)
+AZP_USER=azure-pipelines
+BAZEL_REMOTE_VERSION=2.4.0
+BAZEL_REMOTE_INSTALL_URL="https://github.com/buchgr/bazel-remote/releases/download/v${BAZEL_REMOTE_VERSION}/bazel-remote-${BAZEL_REMOTE_VERSION}-linux-${FULL_ARCH}"
+BAZELISK_VERSION=1.11.0
+BAZELISK_INSTALL_URL="https://github.com/bazelbuild/bazelisk/releases/download/v${BAZELISK_VERSION}/bazelisk-linux-${UNAME_ARCH}"
+export DEBIAN_FRONTEND=noninteractive
+DOCKER_PK=9DC858229FC7DD38854AE2D88D81803C0EBFCD88
+# Hardcoded Github SSH RSA SHA to ensure we download the known key.
+# This must be updated when Github change their public key.
+GITHUB_PK_SHA=uNiVztksCsDhcc0u9e8BujQXVUpKZIDTMczCvj3tD2s
+# https://software.opensuse.org/download/package?package=skopeo&project=devel%3Akubic%3Alibcontainers%3Astable#manualUbuntu
+KUBIC_REPO_URL="https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/xUbuntu_20.04"
+
+alias apt-get="apt-get -qq"
```

Comment:
this doesnt work

Reply:
Did you want to remove or fix it? Asking due to the comment above: "this doesn't work".

Reply:
fixed it